test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -179,6 +179,7 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.cloud_integrations.full_story.org_id (any)',
         // No PII. Just the list of event types we want to forward to FullStory.
         'xpack.cloud_integrations.full_story.eventTypesAllowlist (array)',
+        'xpack.cloud_integrations.gain_sight.org_id (any)',
```

Comment:
Does Gainsight consider the `org_id` public information, or is it more akin to a secret?

Reply:
yes it's public information, it's used in all API requests as url parameter

Reply:
Brilliant, thanks for confirming!